docs/01-programs/01-token-metadata/00-overview.md

```diff
@@ -78,11 +78,26 @@ To safely identify the fungibility of a token — and, thus, the standard that w
 - `NonFungibleEdition`: This is the same as `NonFungible` but the NFT was printed from an Original NFT and, thus, is associated with an Edition account instead of a Master Edition account.
 - `FungibleAsset`: The Mint account is Fungible but has zero decimal places. Having zero decimals means we can treat the token as an asset whose supply is not limited to one. For instance, Fungible Assets can be used in the gaming industry to store resources such as “Wood” or “Iron”.
 - `Fungible`: The Mint account is Fungible and has more than one decimal place. This is more likely going to be a token used as a decentralised currency.
+- `ProgrammableNonFungible`: TODO. See the next section for more information.
 
 You can [read more about these standards here](./token-standard).
 
 ![This image shows three diagrams representing all three fungibility standards. From top to bottom. "NonFungible": Shows a Mint Account with the following attributes "Mint Authority = Edition", "Supply = 1", "Decimals = 0" and "Freeze Authority = Edition". It points to two PDAs, one pointing to a Metadata Account and one pointing to both a Master Edition Account and an Edition Account with a big OR in the middle. "FungibleAsset": Shows a Mint Account with the following attribute: "Decimals = 0". It points to a single PDA which points to a Metadata Account. "Fungible": Shows the same diagram as the "FungibleAsset" diagram but instead of "Decimals = 0", it displays "Decimals > 0" under the Mint Account.](/assets/programs/token-metadata/Token-Metadata-Overview-7.png#radius)
 
+## Programmable NFTs
```

Comment:
The diagram at the start does not include the changes to the metadata account (e.g., `programmable_config`) – not sure if this is intentional.